src/sentry/tasks/groupowner.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import, print_function
+
+from datetime import timedelta
+
+from django.utils import timezone
+
+from sentry.models.groupowner import GroupOwner, GroupOwnerType
+from sentry.tasks.base import instrumented_task
+from sentry.utils.committers import get_serialized_event_file_committers
+
+
+PREFERRED_GROUP_OWNERS = 3
+PREFERRED_GROUP_OWNER_AGE = timedelta(days=1)
+
+
+@instrumented_task(
+    name="sentry.tasks.process_suspect_commits", default_retry_delay=60 * 5, max_retries=5
+)
+def process_suspect_commits(event, **kwargs):
```

Comment:
Pushing the event into rabbit is expensive, as the event has to be serialized and sent to rabbit and then read and deserialized again. Would you be able to use `sentry.eventstore.processing` cache here? That would read from memcache instead saving some memory and network for rabbit.

Reply:
Thanks for this tip and the help on Slack. Implemented a version using the eventstore.